lib/logic/cubit/asset_send_summary/asset_send_summary_state.dart

```diff
@@ -191,7 +192,7 @@ class AssetTransactionSentFailure extends SignedTransactionsLoadedSuccess {
   });
 
   @override
-  List<Object?> get props => [error];
+  List<Object?> get props => [super.props, error];
```

Comment:
Just to be consistent with other code.

```suggestion
  List<Object?> get props => [...super.props, error];
```